### PR TITLE
config template update

### PR DESCRIPTION
**Pull request by Aylr**
- small config template update﻿
- tiny bugfix in data docs due to prior merges

### CODE REVIEW

File: docs/reference/data_context_reference.rst [hunk 1]
```diff
@@ -260,20 +260,16 @@ new directory or use this template:
...
-    # NOTE: GE uses the names of configured `datasources` and `generators` to manage
-    # how `expectations` and other artifacts are stored in the `expectations/` and
-    # `datasources/` folders. If you need to rename an existing `datasource` or
-    # `generator`, be sure to also update the relevant directory names.
 
     config_version: 1
 
     # Datasources tell Great Expectations where your data lives and how to get it.
-    # You can use the CLI command ``great_expectations datasource new`` to help you
+    # You can use the CLI command `great_expectations add-datasource` to help you
```
Comment (eugmandel) [inline docs/reference/data_context_reference.rst]: @Aylr Isn't it the other way?

**Review by eugmandel — DISMISSED** [review]

File: docs/reference/data_context_reference.rst (hunk 1 above)
Comment (jcampbell) [inline docs/reference/data_context_reference.rst]: eyeball check...is the *new* syntax add-datasource?

**Review by jcampbell — APPROVED** [review]
Note one question, but assuming the line is correct, this looks excellent!

File: docs/reference/data_context_reference.rst (hunk 1 above)
Comment (Aylr) [reply docs/reference/data_context_reference.rst]: aaaaack thank you. copypasta the wrong direction

Comment (Aylr) [reply docs/reference/data_context_reference.rst]: Lol, you have both passed my round robin test.

**Pull request merged**